pywatts/core/step_information.py

```diff
@@ -14,8 +14,8 @@ def __init__(self, step: BaseStep, pipeline):
         self.pipeline = pipeline
 
     def __getitem__(self, item: str):
-        from pywatts.core.pipeline_step import PipelineStep
-        if isinstance(self.step, PipelineStep) or len(self.step.targets) > 1:
+        from pywatts.core.step import Step
+        if isinstance(self.step, Step) or len(self.step.targets) > 1:
```

Comment:
Can you check if or `len(self.step.targets) > 1` is necessary by checking if the examples and the unittests work without the second part of the if condition?

Reply:
Not neccessary. I just rebase and push it.